reactor-core/src/test/java/reactor/core/publisher/FluxRetryPredicateTest.java

```diff
@@ -41,19 +42,13 @@ public void sourceNull() {
 		new FluxRetryPredicate<>(null, e -> true);
```

Comment:
The implementation `FluxRetryPredicate` can most likely also be deleted. Please double check remaining relevant tests are either kept in this test class or moved to `FluxRetryWhenTest` as a `@Nested` test maybe?